sources/packages/api/src/services/application/application.service.ts

```diff
@@ -1442,4 +1443,33 @@ export class ApplicationService extends RecordDataModelService<Application> {
       .setParameter("isApplicationArchived", true)
       .execute();
   }
+
+  /**
+   * Get offering request change for a particular application id.
+   * @param applicationId
+   * @returns Offering id and submitted date.
+   */
+  async getOfferingChangeRequestsByApplicationId(
```

Comment:
do we need `Id` in this function name  as where condition is 
![image](https://user-images.githubusercontent.com/77353155/186484056-5ce07d72-edc0-4f49-9601-ea0ef960b4ba.png)